src/__tests__/__snapshots__/index.spec.tsx.snap

```diff
@@ -4,39 +4,27 @@ exports[`Expanded depth nested object static given initial level set to 0, shoul
 "<div class=\\"\\" id=\\"mosaic-provider-react-aria-0-1\\">
```

Comment:
at this point it feels appropriate to remove the snapshots.
Let me explain why I added them in the first place.
About a year ago I was moving the whole logic off to JST. This is the PR that introduced the following snapshots https://github.com/stoplightio/json-schema-viewer/pull/92.
Once JST was founded, I incorporated it over here https://github.com/stoplightio/json-schema-viewer/pull/91/files and these snapshots were very helpful and ensured me that JST didn't have any meaningful impact on the UI.
Besides a few new tests and actual bug fixes (JST did help to fix a number of bugs as well as improved the overall processing of schemas), the snapshots were left untouched.


Now, when we actively work on UI, they don't have any value.

Reply:
done here https://github.com/stoplightio/json-schema-viewer/pull/180/files